web-registry/src/mocks/mocks.ts

```diff
@@ -10,6 +10,8 @@ export interface CreditClass {
   methodologyUrl: string;
   methodologyId: string;
   imgSrc: string;
+  admin?: string;
+  issuers?: string[];
```

Comment:
why do you need this? the admin and issuers are available on chain: https://docs.regen.network/modules/ecocredit/02_state.html#class-info-table

Reply:
I meant to delete this. Thx